test/Microsoft.AspNetCore.SignalR.Tests/HubEndpointTests.cs

```diff
@@ -36,6 +38,106 @@ public class HubEndpointTests
             }
         }
 
+        [Fact]
+        public async Task CanReturnValuesFromMethodsOnHub()
```

Comment:
Should this be split up into multiple tests? One for each of the HubMethods you are checking